integration

```diff
@@ -1533,11 +1533,12 @@
   "zbigniewmotyka/home-assistant-solplanet",
   "zeronounours/HA-custom-component-energy-meter",
   "zhbjsh/homeassistant-ssh",
+  "zheffie/mijn_waterlink",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>